engine/player/player.cpp

```diff
@@ -11733,6 +11733,14 @@ void player_t::create_options()
   add_option( opt_bool( "infinite_runic", resources.infinite_resource[ RESOURCE_RUNIC_POWER ] ) );
   add_option( opt_bool( "infinite_astral_power", resources.infinite_resource[ RESOURCE_ASTRAL_POWER ] ) );
 
+  // Rygelon Dagger / Antumbra
+  add_option( opt_bool( "antumbra.swap", antumbra.swap ) );
+  add_option( opt_float( "antumbra.int_diff", antumbra.int_diff ) );
+  add_option( opt_float( "antumbra.crit_diff", antumbra.crit_diff ) );
+  add_option( opt_float( "antumbra.haste_diff", antumbra.haste_diff ) );
+  add_option( opt_float( "antumbra.mastery_diff", antumbra.mastery_diff ) );
+  add_option( opt_float( "antumbra.vers_diff", antumbra.vers_diff ) );
```

Comment:
```suggestion
  // Rygelon Dagger / Antumbra
  add_option( opt_bool( "antumbra.swap", antumbra.swap ) );
  add_option( opt_float( "antumbra.int_diff", antumbra.int_diff ) );
  add_option( opt_float( "antumbra.crit_diff", antumbra.crit_diff ) );
  add_option( opt_float( "antumbra.haste_diff", antumbra.haste_diff ) );
  add_option( opt_float( "antumbra.mastery_diff", antumbra.mastery_diff ) );
  add_option( opt_float( "antumbra.vers_diff", antumbra.vers_diff ) );
  add_option( opt_float( "antumbra.stam_diff", antumbra.stam_diff ) );
```
Stamina affects DPS for some classes, so it wouldn't hurt to have that too.